packages/main/src/Tokenizer.ts

```diff
@@ -710,18 +932,25 @@ class Tokenizer extends UI5Element {
 	get _selectedTokens() {
 		return this._getTokens().filter(token => token.selected);
 	}
+	get _nMoreListMode() {
+		if (this.readonly || this.disabled) {
+			return "None";
+		}
+
+		return "Delete";
+	}
 
 	get classes(): ClassMap {
```

Comment:
// I suggest you to completely rework this and remove the get classes method
	// base styles on DOM attributes

Reply:
I removed whatever could be replace with just css, rest of it will be removed once we adopt the popover API.